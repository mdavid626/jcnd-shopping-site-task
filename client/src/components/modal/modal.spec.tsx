import { cleanup, render } from '@testing-library/react';
import React from 'react';
import Modal from './modal';

describe('modal', () => {
  afterEach(cleanup);

  it('should render', () => {
    const { asFragment } = render(
      <>
        <Modal>
          <div>content</div>
        </Modal>
        <div id="modal-root" data-testid="modal-root"></div>
      </>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
