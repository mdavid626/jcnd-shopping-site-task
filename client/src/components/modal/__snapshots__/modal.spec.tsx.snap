// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`modal should render 1`] = `
<DocumentFragment>
  <div
    data-testid="modal-root"
    id="modal-root"
  >
    <div
      class="Modal"
    >
      <div>
        content
      </div>
    </div>
  </div>
</DocumentFragment>
`;
